src/main/java/uk/org/openbanking/datamodel/error/OBStandardErrorCodes1.java

```diff
@@ -66,8 +66,7 @@
 
     UK_OBIE_INVALID_CONSENT_STATUS("UK.OBIE.Resource.InvalidConsentStatus"),
 
-    ;
-
+    UK_OBIE_FAILS_CONTROL_PARAMETERS("UK.OBIE.Rules.FailsControlParameters");
```

Comment:
I'm sure I added this last week. Lemme find it!

Reply:
Not sure what's going on here I added this Enum in this PR at line 50 ish
https://github.com/OpenBankingToolkit/openbanking-uk-datamodel/pull/140/files

Reply:
This should be 
UK_OBIE_RULES_FAILS_CONTROL_PARAMETERS("UK.OBIE.Rules.FailsControlParameters"), shouldn't it?

Reply:
Fixed!